File: FangZhiHu/CommonLibKotlin/src/main/java/network/test/sa.java
package network.test;

import android.util.Log;

import java.util.HashMap;
import java.util.Map;
import java.util.Set;

/**
 * Created by tongqinyuan on 2018/3/7.
 */

public class sa {

    public void ss(){
        Map<String,Object> header = new HashMap<>();
        header.put("name","张三");


        if(header != null && header.size() > 0){
            Set<String> strings = header.keySet();
            for (String item : strings){
                Object o = header.get(item);
                Log.e("test",item+"==key==value=="+o);
            }
        }

    }
}

File: FangZhiHu/KotlinOKhttp/src/main/java/com/kotlinokhttp/Sa.kt
package com.kotlinokhttp

import java.security.Key

/**
 * Created by tongqinyuan on 2018/3/6.
 */
class Sa(val value: Int) {

     var ss :String = "妮妮"

    companion object {
        @JvmField
        var string:String  = "哈哈哈";

        @JvmField
        val COMPARATOR: Comparator<Sa> = compareBy<Sa> { it.value }

        var name:String = "中国"
    }
}